apps/e2e/mix.exs

```diff
@@ -34,7 +34,7 @@ defmodule E2E.MixProject do
       {:estuary, in_umbrella: true, only: [:integration]},
       {:flair, in_umbrella: true},
       {:discovery_streams, in_umbrella: true},
-      {:divo, "~> 1.1", only: [:dev, :test, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in e2e, e2e needs a new mix version